Build/build.ps1

```diff
@@ -31,7 +33,62 @@ function Build-One {
 }
 
 Get-ChildItem (Join-Path $PSScriptRoot '..') -Recurse -Include '*.sln' `
-    | ForEach-Object { Build-One $_.FullName }
+    | ForEach-Object { Build-One $_.FullName $False }
+
+$QirProjects = @(
+    #(Join-Path $PSScriptRoot .. samples algorithms chsh-game CHSHGame.csproj),
```

Comment:
I take it commented-out lines correspond to projects  not yet compatible with QIR? That seems like a reasonable exception to the general advice of not leaving commented-out code in the repo, but it may be helpful to have a quick comment explaining that.

Reply:
Yah good idea. Done.